src/main/java/org/fundacionjala/codewars/EANValidator.java

```diff
@@ -0,0 +1,44 @@
+package org.fundacionjala.codewars;
+
+/**
+ * Validate code EAN
+ */
+public class EANValidator {
+
+    public static boolean validate(String value) {
+        boolean isCorrect;
+        int lastValue = Character.getNumericValue(value.charAt(value.length() - 1));
+        int sum = calculateSum(value);
+        isCorrect = calculateResult(lastValue, sum);
+        return isCorrect;
+    }
+
+    private static int calculateSum(String value) {
+        final int mod = 2;
+        int sum = 0;
+        int total;
```

Comment:
this variable should be declared in the for.
Remember, the variables should be declared in the near scope.